### PR TITLE
Convert NetworkDropdownIcon component to JSX

### PR DESCRIPTION
**Pull request by whymarrh**
Refs #6291

This PR replaces hyperscript in the `NetworkDropdownIcon` component with JSX.

### CODE REVIEW

File: ui/app/components/app/dropdowns/components/network-dropdown-icon.js [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
+NetworkDropdownIcon.defaultProps = {
+  backgroundColor: undefined,
+  innerBorder: 'none',
+  diameter: '12',
+  isSelected: false,
+}
+
+NetworkDropdownIcon.propTypes = {
+  backgroundColor: PropTypes.string,
+  loading: PropTypes.any,
```
Comment (Gudahtt) [inline ui/app/components/app/dropdowns/components/network-dropdown-icon.js]:
```suggestion
  loading: PropTypes.boolean,
```
Pretty sure this is always a `boolean`?

File: ui/app/components/app/dropdowns/components/network-dropdown-icon.js [hunk 2]
```diff
@@ -1,47 +1,53 @@
...
           width: `${diameter}px`,
-        },
-      })
+        }} />
+      </div>
     )
 }
+
+NetworkDropdownIcon.defaultProps = {
+  backgroundColor: undefined,
```
Comment (Gudahtt) [inline ui/app/components/app/dropdowns/components/network-dropdown-icon.js]: Hmm. So do you intend to add `require-default-props` at some point? Should `loading` be added to these default props as well? Seems a _little_ inconsistent to add `undefined` defaults for some props but not others. Not that it bothers me - I'm just wondering.

Comment (whymarrh) [reply ui/app/components/app/dropdowns/components/network-dropdown-icon.js]: Yes, loading should be added, that's my bad

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**